libr/core/cmd_anal.c

```diff
@@ -7906,6 +7906,10 @@ static int cmd_anal_all(RCore *core, const char *input) {
 				oldstr = r_print_rowlog (core->print, "Check for objc references");
 				r_print_rowlog_done (core->print, oldstr);
 				cmd_anal_objc (core, input + 1, true);
+				static const char *oldstr = NULL;
+				oldstr = r_print_rowlog (core->print, "Check for vtables");
+				r_core_cmd0 (core, "avrr");
+				r_print_rowlog_done (core->print, oldstr);
```

Comment:
free the oldstr after this line, and dont make it static or even const

Reply:
Actually the `static const char *oldstr = NULL;` is a copy/pasta mistake, i've removed it